.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,53 @@
+name: Pipeline Script
+
+on:
+  workflow_dispatch:
+
```

Comment:
If scripts require .env files or other secrets, these are not handled or injected by the pipeline. Use workflow secrets ???